microsoft/devhomegithubextension
# Update StyleCop

## Summary of the pull request

Updated StyleCop to the latest version to improve compatibility with latest .NET/C# feature.

## References and relevant issues

Follow-up of https://github.com/microsoft/devhome/pull/2162

## Detailed description of the pull request / Additional comments

- Addressed StyleCop violations after the update
- Removed obsolete StyleCop suppressions in GlobalSuppressions.cs

## Validation steps performed

- Changes are mostly spaces and new lines and shouldn't have any impact.
- Verified that the extension build and run as expected when loaded by DevHome.

## PR checklist
